src/css/curCSS.js

```diff
@@ -16,7 +16,10 @@ function curCSS( elem, name, computed ) {
 	computed = computed || getStyles( elem );
 
 	// Support: IE <=9 only
-	// getPropertyValue is only needed for .css('filter') (#12537)
+	//
+	// getPropertyValue is needed for:
+	//   .css('filter') (#12537)
+	//   .css('--customProperty) (#3144)
```

Comment:
Yeah i was looking at the added lines and not thinking about it seeming to be related to IE9. I'd be fine with removing the `// Support:` comment since like you say it's no longer just an IE9 requirement.